package.json

```diff
@@ -1,37 +1,37 @@
-  "version": "1.3.0",
+  "version": "1.4.0",
   "description": "Microservice for a lightweight notification application",
   "author": "Nick Mcally <nick@amida.com>",
-  "main": "index.js",
+  "main": "dist/index.js",
   "private": false,
   "engines": {
-    "node": ">=6.9.1",
-    "npm": ">=2.15.11",
-    "yarn": ">=0.20.3"
+    "node": ">=8.14.0",
+    "npm": ">=6.4.0",
+    "yarn": ">=1.12.0"
   },
   "scripts": {
-    "start": "gulp serve",
-    "serve": "node ./dist/index.js",
-    "migrate": "./node_modules/.bin/sequelize db:migrate",
-    "migrate:undo": "./node_modules/.bin/sequelize db:migrate:undo",
-    "update_uuid": "node dist/tools/addUUID.js",
-    "create_missing_users": "node dist/tools/create_missing_users.js",
-    "create_db": "cross-env NODE_ENV=test node dist/create_db.js",
-    "delete_db": "cross-env NODE_ENV=test node dist/delete_db.js",
-    "start:debug": "cross-env DEBUG=amida-notification-microservice:* yarn start",
     "build": "gulp",
+    "create-push-notifications-service-user": "node dist/tools/createPushNotificationsServiceUser.js",
+    "create_db": "sequelize db:create",
+    "create_missing_users": "node dist/tools/create_missing_users.js",
+    "delete_db": "sequelize db:drop",
+    "jenkins": "cross-env NODE_ENV=test yarn delete_db; cross-env NODE_ENV=test yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test yarn test",
     "lint": "esw -c .eslintrc *.js server config --color",
-    "lint:watch": "yarn lint -- --watch",
-    "beautify": "esw -c .eslintrc *.js server config --color --fix",
-    "precommit": "yarn lint && yarn test",
-    "test": "gulp && yarn create_db && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive || yarn delete_db",
-    "test:watch": "yarn test -- --watch",
-    "test:coverage": "cross-env NODE_ENV=test ./node_modules/.bin/istanbul cover _mocha -- --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive",
-    "test:check-coverage": "yarn test:coverage && istanbul check-coverage",
+    "lint:fix": "yarn lint --fix",
+    "lint:watch": "yarn lint --watch",
+    "migrate": "sequelize db:migrate",
+    "migrate:undo": "sequelize db:migrate:undo",
     "report-coverage": "coveralls < ./coverage/lcov.info",
+    "serve": "node ./dist/index.js",
     "snyk-protect": "snyk protect",
-    "create-push-notifications-service-user": "node tools/createPushNotificationsServiceUser.js"
+    "start": "gulp serve",
+    "start:debug": "cross-env DEBUG=amida-notification-microservice:* yarn start",
+    "test": "cross-env NODE_ENV=test mocha --opts .mocharc",
+    "test:check-coverage": "yarn test:coverage && istanbul check-coverage",
+    "test:coverage": "cross-env NODE_ENV=test istanbul cover _mocha -- --opts .mocharc",
+    "test:watch": "yarn test --watch",
+    "update_uuid": "node dist/tools/addUUID.js"
```

Comment:
This should run `yarn build` aka `gulp` first since it relies on files in the `dist/` folder